flowmachine/flowmachine/features/subscriber/unique_subscribers_from_queries.py

```diff
@@ -0,0 +1,28 @@
+from typing import List
+
+from flowmachine.core.query import Query
+
+
+class UniqueSubscribersFromQueries(Query):
+    """
+    Given a list of queries with a 'subscriber' column, returns a table of unique subscribers.
+    """
+
+    def __init__(self, query_list: List[Query]):
+        self.query_list = query_list
+        super().__init__()
+
+    @property
+    def column_names(self) -> List[str]:
+        return ["subscribers"]
+
+    def _make_query(self):
```

Comment:
This will work fine provided there are at least two queries, but won't de-duplicate if there's only one query, which is not the behaviour a user would intuitively expect.

Could instead do
```
SELECT subscriber
FROM (
    SELECT subscriber FROM ...
    UNION ALL
    ...
) unioned
GROUP BY subscriber
```
which would work for any number of input queries.

Reply:
Without the `ALL` tho, right? You'd sort of hope it resolves to the same plan _anyway_ but I bet it doesn't.

Reply:
Result should be the same with or without the `ALL` if we then do a `GROUP BY`. My thinking was that if there's any difference in the query plans, the `UNION ALL` version would probably be the faster one, but I could be wrong.

Reply:
Wouldn't this lead to the deep nesting we were hoping to avoid by going around query.union()?

Reply:
Shouldn't do - it's only one extra level of nesting, regardless of the number of queries. And it's an extra level of nesting in the SQL, but not in the flowmachine objects, so it shouldn't matter.

Reply:
Ah, I misunderstood your ellipsis. Will sort.